core-primitives/ocall-api/src/lib.rs

```diff
@@ -21,13 +21,12 @@ pub extern crate alloc;
 
 use alloc::vec::Vec;
 use codec::{Decode, Encode};
-use core::fmt::Debug;
 use itp_types::{TrustedOperationStatus, WorkerRequest, WorkerResponse};
 use sgx_types::*;
 use sp_runtime::OpaqueExtrinsic;
 
 /// Trait for the enclave to make o-calls related to remote attestation
-pub trait EnclaveAttestationOCallApi: Clone + Debug + Send + Sync {
+pub trait EnclaveAttestationOCallApi: Clone + Send + Sync {
```

Comment:
Had to remove the `Debug` trait bound on these traits. It would've required a manual implementation of `Debug` when we have complex members like `Arc<MyService>` in the implementation, which IMO is just not worth the effort. `Debug` is fine on data objects, but not on service components.